starters/angular-ngrx-scss/src/app/counter-example/counter-example.component.scss

```diff
@@ -15,13 +17,10 @@
   }
 }
 
-.counter-example__button {
-  margin: 0 4px;
-}
-
 .counter-example__links {
   padding: 0;
   list-style: none;
+  margin: unset;
```

Comment:
The `<ul>` has a default margin that I got rid of here. The other starter I looked at also had no margin if I recall correctly, but that was because they were not using `<ul>`.